fv3core/fv3core/_config.py

```diff
@@ -325,7 +325,7 @@ def from_namelist(cls, namelist: Namelist) -> "DynamicalCoreConfig":
             vtdm4=namelist.vtdm4,
             z_tracer=namelist.z_tracer,
             do_qa=namelist.do_qa,
-            layout=namelist.layout,
+            layout=namelist.layout,  # type: ignore
```

Comment:
Is this type: ignore still needed, now that you fixed the problem with passing lists vs tuples on init?

Reply:
no longer needed, good catch